lib/assembly-image/image.rb

```diff
@@ -114,6 +114,11 @@ def create_jp2(params = {})
       Jp2Creator.create(self, params)
     end
 
+    def srgb?
+      vips_image = Vips::Image.new_from_file path
```

Comment:
can/should we avoid instantiating this all over the file?  make a memoized instance?

Reply:
I think we only do it this once.